components/Typography/src/MDCFontScaler.h

```diff
@@ -0,0 +1,71 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <UIKit/UIKit.h>
+
+// Material's text styles, which are similar, but not quite equivalent to Apple's UIFontTextStyle.
+typedef NSString *_Nonnull MDCTextStyle NS_TYPED_EXTENSIBLE_ENUM;
+
+extern MDCTextStyle MDCTextStyleHeadline1;
+extern MDCTextStyle MDCTextStyleHeadline2;
+extern MDCTextStyle MDCTextStyleHeadline3;
+extern MDCTextStyle MDCTextStyleHeadline4;
+extern MDCTextStyle MDCTextStyleHeadline5;
+extern MDCTextStyle MDCTextStyleHeadline6;
+extern MDCTextStyle MDCTextStyleSubtitle1;
+extern MDCTextStyle MDCTextStyleSubtitle2;
+extern MDCTextStyle MDCTextStyleBody1;
+extern MDCTextStyle MDCTextStyleBody2;
+extern MDCTextStyle MDCTextStyleButton;
+extern MDCTextStyle MDCTextStyleCaption;
+extern MDCTextStyle MDCTextStyleOverline;
+
+/**
+ MDCFontScaler attaches a scaling curve to a UIFont via an associated object on that font instance.
```

Comment:
make this more clear what we are attaching (not a MDCFontScaler)

Reply:
Added clarity.